packages/strapi-plugin-content-manager/admin/src/reducers.js

```diff
@@ -7,6 +8,7 @@ const reducers = {
   [`${pluginId}_main`]: mainReducer,
   [`${pluginId}_listView`]: listViewReducer,
   [`${pluginId}_rbacManager`]: rbacManagerReducer,
+  [`${pluginId}_editViewLayoutManager`]: editViewLayoutManagerReducer,
```

Comment:
WYT of calling this "editView" instead of the long name that tightly couples the reducer key to the component itself?

Reply:
I'd prefer to keep this name as when you use a selector you directly what the reducer is used for.